client/lib/deploy.rs

```diff
@@ -76,31 +76,37 @@ impl From<GetBlockResult> for ListDeploysResult {
 pub(super) enum OutputKind<'a> {
     File {
         path: &'a str,
+        tmp_path: String,
```

Comment:
Since we're generating `tmp_path` using `Path` functionality (see later comment) I'd be more inclined to hold this as a `PathBuf`.

Reply:
Good point